src/components/v2/Modal/styles.ts

```diff
@@ -21,16 +21,20 @@ export const useModalStyles = ({
       width: 80%;
       max-width: 544px;
       border-radius: ${theme.spacing(3)};
-      padding-top: ${theme.spacing(3)};
       background-color: ${theme.palette.background.paper};
+      overflow: scroll;
+      max-height: 100%;
```

Comment:
Nice one 👍

I have two suggestions:
```suggestion
      overflow: auto;
      max-height: max-height: calc(100% - ${theme.spacing(4)});
```

And I'd also change the width property:
```css
width: calc(100% - ${theme.spacing(4)});
```

Doing that means that on mobile on the spacing between the modal and the borders of the screens will always be 16px, which is the spacing used in the designs for the cards:
<img width="479" alt="Screen Shot 2022-03-28 at 10 44 22 AM" src="https://user-images.githubusercontent.com/44675210/160371879-6569bb99-cf4a-47ae-bb67-8379c57d8050.png">

This is what the modal looks like on mobile with these css properties:
<img width="336" alt="Screen Shot 2022-03-28 at 10 39 00 AM" src="https://user-images.githubusercontent.com/44675210/160371911-b16177b2-9daf-4c49-90d1-fcf0fb064b13.png">